readme.md

```diff
@@ -405,6 +414,26 @@ Default: `false`
 
 If `true`, no quoting or escaping of arguments is done on Windows. Ignored on other platforms. This is set to `true` automatically when the `shell` option is `true`.
 
+#### execPath (for `fork` only)
+
+Type: `string`<br>
+Default: `process.execPath`
+
+Define the sub-process executable binary.
```

Comment:
Let's re-use the Node.js documentation: `Executable used to create the child process.`